src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -843,6 +854,9 @@ public void joinConferenceRoom()
                 updateFromRoomConfiguration();
             }
 
+            // set the unique meeting id from the muc configuration.
+            updateMeetingId();
```

Comment:
Can we remove this from here and be getting that on the first time the meetingID is needed in a get method or something. 
The point is we will be slowing down the join conference by doing a disco-info and waiting for the reply Joining is already slow ... :)

Reply:
Good point, thanks damencho!